### PR TITLE
29 implement system to translate user input into a database query to insert new information

### LINKED ISSUE #29: Implement system to translate user input into a database query to insert new information
Points: 5
Get user input from web app and configure it in a way so that it can be made into a database query.

### PR DESCRIPTION
**Pull request by Fahim-A-Karim**
Implemented the ability to click button and add logic gate, in future will implement deletion, custom gates and i/o

### CODE REVIEW

File: digital_logic_simulator/settings.py [hunk 1]
```diff
@@ -21,10 +21,10 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config('SECRET_KEY')
+SECRET_KEY = 'a'
```
Comment (joshuas4) [inline digital_logic_simulator/settings.py]: Again, this settings.py should not be changed. Follow the direcitons in the readme

File: logicsim/views.py [hunk 2]
```diff
@@ -6,6 +6,22 @@
 # Create your views here.
 from logicsim.models import LogicGate
 
+def createGate(request):
```
Comment (joshuas4) [inline logicsim/views.py:9]: This function isn't tested from what it looks like in tests.py

**Review by joshuas4 — CHANGES_REQUESTED** [review]
Add tests for createGate(request) and revert changes to settings.py

File: digital_logic_simulator/settings.py [hunk 3]
```diff
@@ -21,10 +21,10 @@
...
```
Comment (Nguyen-Tommy) [inline digital_logic_simulator/settings.py]: Changes on this file should be reverted.

**Pull request merged**